src/webapp/pages/analysis/steps/1-outliers/OutliersStep.tsx

```diff
@@ -23,19 +20,13 @@ export const OutliersStep: React.FC<PageStepProps> = React.memo(props => {
 
     const runAnalysis = () => {
         runAnalysisOutlier(
```

Comment:
The order of params is not correct. Look at the `runAnalysisOutlier` method in the `useOutlier` hook. It should be: `(algorithm: string, analysisId: Id, sectionId: Id, threshold: string`.

My proposal to avoid this happening to us in the future is to use an object:

```ts
const runAnalysisOutlier = React.useCallback(
        (options: RunAnalysisOptionsProps) => {
        const { sectionId, algorithm, analysisId, threshold } = options;
        // rest of the code
})

type RunAnalysisOptionsProps = { algorithm: string; analysisId: Id; sectionId: Id; threshold: string };
```

and from the component:

```tsx
const runAnalysis = () => {
        runAnalysisOutlier({
            algorithm: qualityFilters.algorithm,
            analysisId: analysis.id,
            sectionId: section.id,
            threshold: qualityFilters.threshold,
        });
    };
```

What do you think?

Reply:
Good idea, Eduardo. This is also more readable and maintainable